### PR TITLE
Allow specification of the intake_status

### PR DESCRIPTION
**Pull request by SquarePlayn**
## Proposed changes

This PR adds support for (optionally) specifying the `intake_status` when fetching transactions or a transaction summary from the `Datasets`. If the function call to fetch these does not specify one, a class static default is used. 

The class static default allows for example to set which intake status is to be used before having some other piece of code instantiate the `Datasets` to make it use the specified intake status.

Additionally, a bug is fixed where in fetching the scopes the new `name_human_times_volume` field had nothing specified.

All changes are backward compatible with older versions of the dataset service.

Fixes [AB#273](https://dev.azure.com/jdokter/973dd796-5b61-4e4a-a52b-ab66f88cef8a/_workitems/edit/273)

## Types of changes
- [x] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### Unit testing
None

### Manual testing
Try out fetching different intake statuses, on the latest dataset service (which allows intake status specification) and on an older one without it, for summary and transaction fetching.

### CODE REVIEW

File: src/pricecypher/models/datasets.py [hunk 1]
```diff
@@ -15,6 +15,7 @@ class Scope:
     representation: Optional[str] = field(compare=False)
     name_dataset: str = field(compare=False)
     name_human: Optional[str] = field(compare=False)
+    name_human_times_volume: Optional[str] = field(compare=False)
```
Comment (marijnvanderhorst) [inline src/pricecypher/models/datasets.py:18]: Might be good to also add the `volume_multiplied_constant` to the ScopeConstantTransaction model

File: src/pricecypher/datasets.py [hunk 2]
```diff
@@ -100,23 +106,35 @@ def get_scope_values(self, dataset_id, scope_id, bc_id='all'):
             .scope_values(scope_id)
         )
 
-    def get_transaction_summary(self, dataset_id, bc_id='all'):
+    def get_transaction_summary(self, dataset_id, bc_id='all', intake_status=None):
```
Comment (marijnvanderhorst) [inline src/pricecypher/datasets.py:109]: Would it be a good idea to use some kind of enum for the possible statuses that can be passed, or perform a check on the given value?

**Review by marijnvanderhorst — COMMENTED** [review]
Looks good, 2 small questions

File: src/pricecypher/datasets.py (hunk 2 above)
Comment (SquarePlayn) [reply src/pricecypher/datasets.py:109]: Might be nice to add at some point, but currently I think this is fine. If you input invalid data, the dataset service will tell you 422: the given data was invalid (tho not what exactly was invalid)

File: src/pricecypher/models/datasets.py (hunk 1 above)
Comment (SquarePlayn) [reply src/pricecypher/models/datasets.py:18]: Added

**Review by marijnvanderhorst — APPROVED** [review]
Thanks looks good 👍

**Pull request merged**